usecases/auth/authorization/rbac/model.go

```diff
@@ -104,9 +104,15 @@ func Init(authConfig config.APIKey, policyPath string) (*casbin.SyncedCachedEnfo
 	enforcer.AddNamedMatchingFunc("g", "regexMatch", casbinutil.RegexMatch)
 
 	// add pre existing roles
-	for name, verbs := range builtInRoles {
-		if _, err := enforcer.AddNamedPolicy("p", name, "*", verbs, "*"); err != nil {
-			return nil, fmt.Errorf("add policy: %w", err)
+	for name, permission := range builtInRoles {
```

Comment:
**issue (if-minor):**  builtin roles has to be for all resource , no need to ad permissions permissons